src/Halogen/HTML/Core.purs

```diff
@@ -70,17 +70,18 @@ text :: forall p i. String -> HTML p i
 text = HTML <<< VDom.Text
 
 -- | A smart constructor for HTML elements.
-element :: forall p i. VDom.ElemName -> Array (Prop i) -> Array (HTML p i) -> HTML p i
-element = coe (\name props children -> VDom.Elem (VDom.ElemSpec Nothing name props) children)
+element :: forall p i. Maybe VDom.Namespace -> VDom.ElemName -> Array (Prop i) -> Array (HTML p i) -> HTML p i
+element ns =
+  coe (\name props children -> VDom.Elem (VDom.ElemSpec ns name props) children)
   where
```

Comment:
Yeah, I would say this is a breaking change. We could hold off releasing it properly until we do the 0.11 update and do them together then?

Also, minor nitpick, but I'd prefer we used `NS` in the names rather than `Ns` :smile: